### PR TITLE
k8s 1.25.0

### PR DESCRIPTION
**Pull request by soltysh**
Replaces https://github.com/openshift/origin/pull/27376

### CODE REVIEW

File: pkg/monitor/backenddisruption/disruption_backend_sampler_test.go [hunk 1]
```diff
@@ -84,7 +84,8 @@ func TestBackendSampler_checkConnection(t *testing.T) {
 				connectionType:        NewConnectionType,
 				path:                  "/302-bad-response",
 			},
-			wantErr: true,
+			// TODO: should this be error when Location header is not set?
+			wantErr: false,
```
Comment (dgoodwin) [inline pkg/monitor/backenddisruption/disruption_backend_sampler_test.go:88]: This is passing on other PRs I think, any idea what's in here that's causing this? Not super familiar with the sampler code yet.

Comment (soltysh) [reply pkg/monitor/backenddisruption/disruption_backend_sampler_test.go:88]: Looks like this is coming from go 1.18 -> 1.19 change, apparently missing `Location` header is acceptable.

Comment (soltysh) [reply pkg/monitor/backenddisruption/disruption_backend_sampler_test.go:88]: Yup, found it here: https://github.com/golang/go/commit/a41763539c7ad09a22720a517a28e6018ca4db0f

**Comment by soltysh** [comment]
/hold
so we don't merge the temporary last commit
/test e2e-azure

**Comment by soltysh** [comment]
/hold cancel
all is updated with latest o/k state

**Comment by soltysh** [comment]
/payload-job periodic-ci-openshift-release-master-ci-4.12-upgrade-from-stable-4.11-e2e-azure-sdn-upgrade

**Comment by soltysh** [comment]
/payload-job periodic-ci-openshift-release-master-ci-4.12-upgrade-from-stable-4.11-e2e-azure-sdn-upgrade

**Comment by kikisdeliveryservice** [comment]
/payload-job periodic-ci-openshift-release-master-ci-4.12-upgrade-from-stable-4.11-e2e-aws-ovn-upgrade

**Comment by kikisdeliveryservice** [comment]
/payload-job periodic-ci-openshift-release-master-ci-4.12-e2e-aws-sdn-serial

**Comment by soltysh** [comment]
Manually adding lgtm label back after adding the last commit fixing image ecosystem tests (see last commit)

**Pull request merged**